src/main/java/io/confluent/connect/jdbc/TableQuerier.java

```diff
@@ -107,4 +121,11 @@ public int compareTo(TableQuerier other) {
       return this.name.compareTo(other.name);
     }
   }
+
+  private boolean exceptionIndicatesStalePreparedStatement(Connection db, SQLException e) throws SQLException {
+    // On PostgreSQL, queries with wildcards will be invalidated on schema changes
+    // and throw this error message.
+    return e.getMessage().contains("cached plan must not change result type") &&
```

Comment:
I took a look at the error code returned in the SQLException, and it looked to be zero. I'll see if maybe I can cast the exception to a PSQLException to get better info.